sdv/single_table/copulas.py

```diff
@@ -323,25 +324,53 @@ def _rebuild_correlation_matrix(cls, triangular_correlation):
 
         return cls._get_nearest_correlation_matrix(correlation).tolist()
 
-    def _rebuild_gaussian_copula(self, model_parameters):
+    def _rebuild_gaussian_copula(self, model_parameters, default_params=None):
         """Rebuild the model params to recreate a Gaussian Multivariate instance.
 
         Args:
             model_parameters (dict):
                 Sampled and reestructured model parameters.
+            default_parameters (dict):
+                Fall back parameters if sampled params are invalid.
 
         Returns:
             dict:
                 Model parameters ready to recreate the model.
         """
+        if default_params is None:
+            default_params = {}
+
         columns = []
         univariates = []
         for column, univariate in model_parameters['univariates'].items():
             columns.append(column)
             if column in self._numerical_distributions:
-                univariate['type'] = self._numerical_distributions[column]
+                univariate_type = self._numerical_distributions[column]
+            else:
+                univariate_type = self.get_distribution_class(self.default_distribution)
+
+            univariate['type'] = univariate_type
+            model = univariate_type.MODEL_CLASS
+            if hasattr(model, '_argcheck'):
+                to_check = {
+                    parameter: univariate[parameter]
+                    for parameter in inspect.signature(model._argcheck).parameters.keys()
+                    if parameter in univariate
+                }
+                if not model._argcheck(**to_check):
+                    if column in default_params.get('univariates', []):
```

Comment:
what is the situatiom where the column won't have default parameters?

Reply:
If we load in an old synthesizer and try to sample from it, or in the single table case where we don’t need them.